signer/tests/integration/contracts.rs

```diff
@@ -54,23 +73,26 @@ impl AsContractDeploy for SbtcBootstrapContract {
         include_str!("../../../contracts/contracts/sbtc-bootstrap-signers.clar");
 }
 
-pub struct SignerKeyState {
-    pub state: SignerStxState,
+pub struct SignerStxState {
+    pub wallet: SignerWallet,
     pub keys: [Keypair; 3],
+    pub stacks_client: &'static StacksClient,
+}
+
+fn make_signatures(tx: &StacksTransaction, keys: &[Keypair]) -> Vec<RecoverableSignature> {
+    keys.iter()
+        .map(|kp| stacks::wallet::sign_ecdsa(tx, &kp.secret_key()))
+        .collect()
 }
 
 /// Deploy an sBTC smart contract to the stacks node
-async fn deploy_smart_contract<T>(state: &SignerKeyState, client: &StacksClient, deploy: T)
+async fn deploy_smart_contract<T>(signer: &SignerStxState, deploy: T)
```

Comment:
Not really related to this PR, but it might be nicer if this is a method on the `SignerStxState` struct rather than an independent function.